packages/document/docs/zh/guide/basic/deploy.mdx

```diff
@@ -162,3 +162,23 @@ jobs:
 #### 配置自定义域名
 
 如果你想要将你的站点绑定到自己的域名上，那么你可以在 `Netlify` 的 `Domain management` 栏目中进行配置，具体的配置方法可以参考 [Netlify 官方文档](https://docs.netlify.com/domains-https/custom-domains/)。
+
+### 通过 Kinsta 部署
+
+您可以在 [Kinsta](https://kinsta.com/static-site-hosting/) 上部署你的 Rspress 网站。
+
+1. 登录或创建账户，来查看你的 [MyKinsta](https://my.kinsta.com/) 面板。
+
+2. 通过 Git 提供商授权 Kinsta。
+
+3. 从左侧边栏选择**静态网站**，然后点击**添加网站**。
```

Comment:
The keyword could be reserved because the Kinsta platform does not have chinese version.

Reply:
fixed